tests/integration_test.go

```diff
@@ -205,6 +207,7 @@ func TestSuccess(t *testing.T) {
 				continue
 			}
 			if status != types.CompletedSuccess {
+				cancel()
```

Comment:
I'm attempting to speed up the failure when we already have an error by adding these `cancel()` calls.  The taker was getting Refunded here, while the Maker was tied up for 40 seconds locking his Monero, so we'd have to wait for the Maker to hit his timeout clause before the test could synchronize and print out the error.